src/IFeatureManager.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ITargetingContext } from "./common/ITargetingContext";
+import { Variant } from "./variant/Variant";
+
+export interface IFeatureManager {
+    /**
+     * Get the list of feature names.

```

Comment:
Somewhere you used the third person singular, while in others, you did not.

Reply:
I use `Get...` instead of `Gets..." here. But for the properties of options, I use "It gets/specifies ..." if starting with an "It".

Those copied from dotnet are using third person singular for verbs I guess. ; ) Yes, they should be updated to be consistent.